src/views/subscribe/subscribe.js

```diff
@@ -0,0 +1,235 @@
+/** @jsx jsx */
+import { jsx, Styled } from 'theme-ui'
+import React from 'react'
+import "./style.css"
+import { Container } from '@theme-ui/components'
+import { useFormik } from 'formik';
+import ReactModal from 'react-modal'
+import { useState } from 'react'
+
+const Subscribe = () => {
+  const tag = `OpenEBS OF Newsletter`
+
+  const [showModal, setModal] = useState(false);
+  const [isSuccess, setIsSuccess] = useState(false);
+
+  const GATSBY_NEWSLETTER_BACKEND_URL =
+    process.env.GATSBY_NEWSLETTER_BACKEND_URL ? process.env.GATSBY_NEWSLETTER_BACKEND_URL : 'http://localhost:3000/mayadata-newsletter/openEBS-newsletter/';
```

Comment:
please don't hard code  url `http://localhost:3000/mayadata-newsletter/openEBS-newsletter/`. 
it is better to put it in `.env` file.